src/components/GraphHeartbeats.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Paper } from 'material-ui'
+import Divider from 'material-ui/Divider'
+import { BarChart, Bar, Cell } from 'recharts'
+import Theme from '../../style/theme'
+
+class GraphHeartbeats extends React.PureComponent {
+  static getColor(amt) {
+    if (amt > 1) {
+      return '#00E200'
+    }
+    if (amt === 0) {
+      return '#707070'
+    }
+    let r = Math.floor(((((1 - amt) / 0.5) * 228) + 228) / 2)
+    let g = Math.floor((amt / 0.5) * 228)
+    const b = 0
+    if (amt > 0.5) {
+      g = 228
+    } else {
+      r = 228
+    }
+    return `#${r.toString(16).padStart(2, '0')}${g.toString(16).padStart(2, '0')}${b.toString(16).padStart(2, '0')}`
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      activeIndex: 0,
+      summaryData: {}
+    }
+    this.processData()
```

Comment:
You don't want calculations and processing done in the constructor. Move it to `componentDidMount` (I noticed in some components you are doing it in componentDidMount).